client/web/src/repo/tree/TreePage.module.scss

```
@@ -41,3 +59,102 @@
     width: 100%;
     max-width: var(--media-xl);
 }
+
+.readme-container {
+    max-height: 30rem;
+    overflow: hidden;
+    position: relative;
+    margin-top: 1rem;
+    margin-left: -0.25rem;
+}
+
+.readme-fader {
+    position: absolute;
+    bottom: 0;
+    width: 100%;
+    height: 2.5rem;
+    transition: ease-in;
+    background-image: linear-gradient(transparent, 75%, var(--code-bg));
+}
+
+.readme-fader-invisible {
+    visibility: hidden;
+}
+
+.readme {
+    padding: 0 0.25rem;
+}
+
+.readme-more {
+    margin-top: 0;
```

IMO this needs a touch of padding:

```suggestion
    margin-top: 0.5rem;
```